SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -1,20 +1,205 @@
+        static let underestimateText = "Занизить"
+        static let shareButtonTitleText = "Похвастаться"
+        static let infoActivityViewText = "Купил ведро -"
+        static let buiButtonAlertDefaultText = "Ок"
+        static let underestimateNumber: CGFloat = 70
+        static let underestimateRange: ClosedRange<CGFloat> = 0...100
+        static let spacingVStack: CGFloat = 15
+        static let showroomNameTextOffset: CGFloat = 10
+        static let spacingButton: CGFloat = 50
+        static let showroomNameTextFontSize: CGFloat = 30
+        static let buiButtonFontSize: CGFloat = 15
+        static let buiButtonWidthNumber: CGFloat = 90
+        static let buiButtonHeightNumber: CGFloat = 25
+        static let buiButtonCornerRadiusNumber: CGFloat = 10
+        static let carImageWidthNumber: CGFloat = 350
+        static let carImageHeightNumber: CGFloat = 300
+        static let carImageCornerRadiusNumber: CGFloat = 50
+        static let zeroNumber = 0
+        static let shareButtonTitleFontNumber: CGFloat = 15
+        static let shareButtonWidthNumber: CGFloat = 130
+        static let shareButtonHeightNumber: CGFloat = 25
+        static let shareButtonCornerRadiusNumber: CGFloat = 10
+    }
+    
+    // MARK: - Public Properties
+
     var body: some View {
-        VStack {
+        NavigationView {
+            VStack(spacing: Constants.spacingVStack) {
+                showroomNameText.offset(y: Constants.showroomNameTextOffset)
+                carImage
+                carPicker
+                Form {
+                    priceText
+                    enginePowerText
+                    toningText
+                    steeringWheelText
+                    salonPicker
+                    HStack {
+                        underestimateText
+                        underestimateSlier
+                    }
+                }
+                HStack(spacing: Constants.spacingButton) {
+                    buyButton
+                    shareButton
+                }
+            }
         }
+    }
+    
+    // MARK: Private Properties
+    
+    private let cars = [
+        Car(name: "Пятера",
+            imageName: "11",
+            toning: "Аквариум",
+            enginePower: 74,
+            price: 25000),
+        Car(name: "Не пешком",
+            imageName: "22",
+            toning: "Гаишники попутают",
+            enginePower: 82,
+            price: 30000),
+        Car(name: "Дедушкина",
+            imageName: "33",
+            toning: "Аквариум",
+            enginePower: 123,
+            price: 15000),
+        Car(name: "Феррари",
+            imageName: "44",
+            toning: "Аквариум",
+            enginePower: 162,
+            price: 100000)
+    ]
+    
+    @State private var selectionCarPicker = Constants.zeroNumber
+    @State private var selectionSalonPicker = Constants.zeroNumber
+    @State private var isShare = false
+    @State private var isBuy = false
```

Comment:
```suggestion
    @State private var isBuy = false
```
isBuyShown ? isBuyPressed?